versioneer.py

```diff
@@ -398,7 +397,7 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False, env=
                 [c] + args, cwd=cwd, env=env, stdout=subprocess.PIPE, stderr=(subprocess.PIPE if hide_stderr else None)
             )
             break
-        except EnvironmentError:
+        except OSError:
```

Comment:
Change of error from EnvironmentError to OSError. According to the [Built-in Exceptions docs](https://docs.python.org/3/library/exceptions.html#OSError.filename2), EnvironmentError was merged into OSError in 3.3